configure.in

```diff
@@ -2371,8 +2371,8 @@ AC_CACHE_CHECK(
   if test "$pr_cv_func_facl" = "none"; then
     old_ldflags=$LDFLAGS
     old_libs=$LIBS
-    new_ldflags=`echo "$LDFLAGS" | sed -e 's/-L\$(top_srcdir)\/lib//g'`
-    LDFLAGS="$new_ldflags"
+    new_ldflags=`echo "$LDFLAGS" | sed -e 's/-Wl\.*\/lib //g'`
+    LDFLAGS="-L\$(top_builddir)/lib $new_ldflags"
```

Comment:
Fortunately, this is the only place affected by this issue (and thus just detection of POSIX ACL support on Linux):
```
$ grep top_srcdir configure.in
LDFLAGS="-Wl,-L\$(top_srcdir)/lib,-L\$(top_builddir)/lib $LDFLAGS"
      MAIN_LDFLAGS="-L\$(top_srcdir)/lib/libltdl -dlopen self -export-dynamic"
    new_ldflags=`echo "$LDFLAGS" | sed -e 's/-L\$(top_srcdir)\/lib//g'`
```